snitun/multiplexer/message.py

```diff
@@ -22,7 +23,7 @@
 class MultiplexerChannelId:
     """Represent a channel ID aka multiplexer stream."""
 
-    bytes: bytes = attr.ib(default=attr.Factory(lambda: os.urandom(16)), eq=True)
+    bytes: "bytes" = attr.ib(default=attr.Factory(lambda: os.urandom(16)), eq=True)
```

Comment:
Because its shadowing a built-in name, it needs to be quoted or pyright can't figure it out